tinygrad/renderer/cstyle.py

```diff
@@ -256,6 +256,7 @@ class CUDALanguage(CStyleLanguage):
                  UnaryOps.EXP2: lambda x,dtype: f"exp2({x})" if dtype != dtypes.half else f"hexp2({x})"}
   half_prekernel = """
     #include <cuda_fp16.h>
+    #include <cuda_bf16.h>
```

Comment:
we need a `dtypes_prekernel`

Reply:
to dynamically select the header files? I thought additional include would be more simple